app/team.go

```diff
@@ -1188,3 +1189,15 @@ func (a *App) RemoveTeamIcon(teamId string) *model.AppError {
 
 	return nil
 }
+
+func (a *App) UpdateTeamPrivacy(oldTeam *model.Team, user *model.User) (*model.Team, *model.AppError) {
+	team, err := a.UpdateTeam(oldTeam)
+	if err != nil {
+		return team, err
+	}
+	messageWs := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_TEAM_CONVERTED, team.Id, "", "", nil)
+	messageWs.Add("team_id", team.Id)
+	a.Publish(messageWs)
+
+	return team, nil
+}
```

Comment:
This works fine but the internal implementation is for general purpose of patching any field of the team.  I'd prefer to narrow down its scope.  I think it's best to do it similar to `UpdateTeamScheme`.